nodepool/config/templates/cluster.yaml

```diff
@@ -46,9 +46,15 @@ availabilityZone: {{.AvailabilityZone}}
 # ARN of the KMS key used to encrypt TLS assets.
 kmsKeyArn: "{{.KMSKeyARN}}"
 
-# Number of worker nodes to create
+# Number of worker nodes to create, for more control use `workerASG` and do not use this setting
 #workerCount: 1
 
+# Auto Scaling Group definition for workers. If only `workerCount` is specified, min and max will be the set to that value and `rollingUpdateMinInstancesInService` will be one less.
+# workerASG:
+#   minSize: 1
+#   maxSize: 3
+#   rollingUpdateMinInstancesInService: 2
```

Comment:
Could we adapt to the new `worker:` key [I've recently added for spot fleets](https://github.com/coreos/kube-aws/blob/46330082520d8307e7062b9e86f34c929008de2b/nodepool/config/templates/cluster.yaml#L79) so that we can have more consistency/distinction between 2 deployment strategy: auto scaling group and spot fleet?

```yaml
#worker:
#  # Auto Scaling Group definition for workers. If only `workerCount` is specified, min and max will be the set to that value and `rollingUpdateMinInstancesInService` will be one less.
#  autoScalingGroup:
#    minSize: 1
#    maxSize: 3
#      rollingUpdateMinInstancesInservice: 2
#  # Spot Fleet definitions for workers
#  # You may specify autoScalingGroup or spotFleet, but not both.
#  # Spot Fleet support is still experimental. It can be changed in backward-incompatible ways
#  spotFleet:
#  *snip*
```